.github/workflows/build.yml

```diff
@@ -15,7 +15,15 @@ permissions:
   contents: read
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}-reusable
+  group: >-
+    ${{
+      github.workflow
+    }}-${{
+      github.ref_name
+    }}-${{
+      github.event.pull_request.number
+      || github.sha
+    }}
```

Comment:
Remove whitespace
```suggestion
  group: ${{ github.workflow }}-${{ github.ref_name }}-${{ github.event.pull_request.number || github.sha }}-reusable
```

Reply:
It was me who asked for wrapped lines FTR.

Also, we decided to drop the reusable suffix. We talked about it with Cam a lot yesterday.

Reply:
I don't think wrapped lines meaningfully help here, and because of the interpolation markers it just adds more visual clutter.

Fine with keeping or removing `-reusable`, it wasn't mentioned in the PR so I thought it was unintentional.